Tools/performance_tests/run_automated.py

```diff
@@ -1,15 +1,33 @@
-#!/usr/common/software/python/2.7-anaconda-4.4/bin/python
-
 import os, sys, shutil, datetime, git
 import argparse, re, time, copy
 import pandas as pd
 from functions_perftest import store_git_hash, get_file_content, \
-                               run_batch_nnode, extract_dataframe
+    run_batch_nnode, extract_dataframe
+
+# Get name of supercomputer and import configuration functions from 
+# machine-specific file
+if os.getenv("LMOD_SYSTEM_NAME") == 'summit':
+    machine = 'summit'
+    from summit import executable_name, process_analysis, \
+        get_config_command, time_min, get_submit_job_command, \
+        get_batch_string, get_run_string, get_test_list
+if os.getenv("NERSC_HOST") == 'cori':
+    machine = 'cori'
+    from cori import executable_name, process_analysis, \
+        get_config_command, time_min, get_submit_job_command, \
+        get_batch_string, get_run_string, get_test_list
 
 # typical use: python run_automated.py --n_node_list='1,8,16,32' --automated
 # Assume warpx, picsar, amrex and perf_logs repos ar in the same directory and
 # environment variable AUTOMATED_PERF_TESTS contains the path to this directory
```

Comment:
does it make sense to define a default on `AUTOMATED_PERF_TESTS` or throw if it is unset? Otherwise it will start looking absolutely in `/` according to the logic below.